text/0000-native-web-component-support.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Native Web Component Support

```

Comment:
To be more precise can we rename this proposal to Native Custom Element support.

```suggestion
title: Native Custom Element Support
```